chapters/inheritance.tex

```diff
@@ -564,6 +564,76 @@ \subsection{Final Element Modification Prevention}\label{final-element-modificat
 \end{lstlisting}
 \end{example}
 
+\subsection{Removing Modifiers -- break}\label{removing-modifiers-break}
+Modifications may contain the special keyword \lstinline!break! instead of an expression.
```

Comment:
From the text in README.md, the proposal was intended for the removal of inherited modifications. From this sentence, `break` can be used to remove any modification, not just in an extends clause. The examples only show the use in extends clauses and some other text in the rest of the section assumes that we are dealing only with modifications in the inherited models. Should `break` be restricted to extends clauses?

Reply:
I don't see that restricting this `break` to extends-clauses is meaningful, even if I see that it is the primary use.
The reason is that it can be nested so if we didn't allow:
```
model A
   C c(x=break);
end A;
```
one could do:
```
model A2
  C c;
end A2;
model A=A2(c(x=break));
```
The selective model extension (MCP0039) break is different, since it cannot be nested.